composer.json

```diff
@@ -23,8 +23,8 @@
     "require-dev": {
         "zendframework/zend-config": "^2.6",
         "zendframework/zend-servicemanager": "^2.7.5 || ^3.0.3",
-        "zendframework/zend-crypt": "^2.6",
-        "phpunit/phpunit": "^6.0.8 || ^5.7.15",
+        "zendframework/zend-crypt": "^2.6 || ^3.0",
+        "phpunit/phpunit": "^7.0 || ^6.0.8 || ^5.7.15",
```

Comment:
This is dev dependency and this is a testing framework, we can use here just the latest version - `^7.1.4` as that repository supports only PHP 7.1+.